contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -279,6 +279,19 @@ def _execute(self, filter_string: str = "", expected_track_count=None):
             wait_for_condition(lambda: len(self._find_tracks()) == expected_track_count)
 
 
+class ZoomOut(CaptureWindowE2ETestCaseBase):
+    """
+    Zooms the capture view out all the way so that the entire capture is visible.
+    """
+
+    def _execute(self):
+        logging.info('Zooming out the capture window')
+        self._time_graph.click_input()
+        # This should be enough so that we can zoom all the way out from any zoom level with any capture duration.
+        for i in range(200):
+            send_keys("s")
+
+
 class Capture(E2ETestCase):
```

Comment:
Just to be sure: There are only formatting changes in this class, is this expected?